mentorship ios/Managers/NetworkManager.swift

```diff
@@ -11,18 +11,20 @@ import Combine
 
 struct NetworkManager {
     static var responseCode: Int = 0
-    
-    static func callAPI<T: Decodable>(urlString: String, httpMethod: String, uploadData: Data, token: String = "") -> AnyPublisher<T, Error> {
+
+    // swiftlint:disable:next all
+    static func callAPI<T: Decodable>(urlString: String, httpMethod: String, uploadData: Data, token: String = "", cachePolicy: NSURLRequest.CachePolicy = .useProtocolCachePolicy) -> AnyPublisher<T, Error> {
```

Comment:
Break up function declaration into multiline and remove swiftlint ignore

Reply:
Read up on Line Wrapping here https://google.github.io/swift/#function-declarations